### PR TITLE
updated aggregation service

### PR DESCRIPTION
**Pull request by chinmayic**
Added new APIs - AggregationSource and Aggregate
Removed - Aggregation service add and remove actions

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -165,52 +161,45 @@ Without CA certificate, curl fails to verify that HTTP connections are secure an
...
-       
-{path} is where you have generated certificates during the deployment of the resource aggregator.
-	
-    
+    {path} is where you have generated certificates during the deployment of the resource aggregator.
 
 **2.** If you are running curl commands on a different server, perform the following steps to provide the rootCA.crt file.
       
-  a. Navigate to `~/ODIM/build/cert_generator` on the server where the resource
+      a. Navigate to `~/ODIM_v1.0/configuration/Odim/certificates` on the server where the resource
```
Comment (rahulroshan-kachchap) [inline docs/README.md]: ODIM_v1.0 is not part of upstream code.

File: docs/README.md [hunk 2]
```diff
@@ -643,7 +630,7 @@ curl -i POST \
 
 ```
 {
-"UserName": "abc",
+"Username": "abc",
```
Comment (rahulroshan-kachchap) [inline docs/README.md]: This is old convention

File: docs/README.md [hunk 3]
```diff
@@ -1214,7 +1201,7 @@ curl -i POST \
    -H 'Authorization:Basic {base64_encoded_string_of_[username:password]}' \
    -H "Content-Type:application/json" \
    -d \
-'{"UserName":"{username}","Password":"{password}","RoleId":"{roleId}"}
+'{"Username":"{username}","Password":"{password}","RoleId":"{roleId}"}
```
Comment (rahulroshan-kachchap) [inline docs/README.md:1209]: old convention

File: docs/README.md [hunk 4]
```diff
@@ -1241,7 +1228,7 @@ curl -i POST \
 
 ```
 { 
-   "UserName":"monitor32",
+   "Username":"monitor32",
```
Comment (rahulroshan-kachchap) [inline docs/README.md:1236]: same as above

File: docs/README.md [hunk 5]
```diff
@@ -1578,84 +1577,85 @@ x-frame-options:sameorigin
...
-      ​      "PreferredAuthType":"BasicAuth(or)XAuthToken",
-      ​      "PluginType":"Compute"      ​
-   }   ​
+{
+   "HostName":"{plugin_host}:45003",
+   "UserName":"admin",
+   "Password":"Plug!n12$4",
+   "Links":{
+      "Oem":{
+         "PluginID":"ILO",
```
Comment (rahulroshan-kachchap) [inline docs/README.md]: ILO is not part of upstream code

**Review by rahulroshan-kachchap — CHANGES_REQUESTED** [review]
Please review the files again. I believe the file README.md is of previous version.

**Review by rahulroshan-kachchap — APPROVED** [review]

**Comment by shivacharanms** [comment]
README.md file for aggregation service also should be changed here

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**